src/utils.py

```diff
@@ -13,6 +13,13 @@
 import pandas as pd
 import re
 
+import collections
+
+try:
+    collectionsAbc = collections.abc
+except AttributeError:
+    collectionsAbc = collections
```

Comment:
what's the use case for the try n except here?

Reply:
Fallback version for this library.
Depends on the python version